src/ui/common/src/components/DataPreviewer/index.tsx

```diff
@@ -20,7 +20,11 @@ type Props = {
   dataTableHeight?: string;
 };
 
-export const DataPreviewer: React.FC<Props> = ({ previewData, error, dataTableHeight }) => {
+export const DataPreviewer: React.FC<Props> = ({
```

Comment:
kinda strange that your linter is picking this up. I just worked on this file yesterday and linted before pushing. Let me verify some things on my end and see if there's any issues.

Reply:
okay, looks like this is correct. ran lint and this file is being changed as well :)
```
diff --git a/src/ui/common/src/components/DataPreviewer/index.tsx b/src/ui/common/src/components/DataPreviewer/index.tsx
index 1df3ba07..68d84c99 100644
--- a/src/ui/common/src/components/DataPreviewer/index.tsx
+++ b/src/ui/common/src/components/DataPreviewer/index.tsx
@@ -20,7 +20,11 @@ type Props = {
   dataTableHeight?: string;
 };
 
-export const DataPreviewer: React.FC<Props> = ({ previewData, error, dataTableHeight }) => {
+export const DataPreviewer: React.FC<Props> = ({
+  previewData,
+  error,
+  dataTableHeight,
+}) => {
   if (!previewData) {
     return null;
   }
@@ -167,4 +171,4 @@ export const DataPreviewer: React.FC<Props> = ({ previewData, error, dataTableHe
   );
 };
 
-export default DataPreviewer;
\ No newline at end of file
+export default DataPreviewer;

```